azure/table_azure_storage_blob.go

```diff
@@ -402,17 +419,17 @@ func listStorageBlobs(ctx context.Context, d *plugin.QueryData, h *plugin.Hydrat
 
 	for item := range blobCh {
 		for _, data := range item {
-			d.StreamLeafListItem(ctx, &blobInfo{data.Blob, data.Name, account.Name, data.Container, account.ResourceGroup, &subscriptionID, account.Account.Location, data.IsSnapshot})
+			d.StreamLeafListItem(ctx, &blobInfo{data.Blob, data.Name, accountName, data.Container, resourceGroup, &subscriptionID, region, data.IsSnapshot})
```

Comment:
Use `StreamListItem`
```suggestion
			d.StreamListItem(ctx, &blobInfo{data.Blob, data.Name, accountName, data.Container, resourceGroup, &subscriptionID, region, data.IsSnapshot})
```